test/jdk/java/net/DatagramSocket/DatagramSocketMulticasting.java

```diff
@@ -326,11 +341,11 @@ static void testSendNoReceive(DatagramSocket s, InetAddress group) throws IOExce
         System.out.println("testSendNoReceive");
 
         // outgoing multicast interface needs to be set
-        assertTrue(s.getOption(IP_MULTICAST_IF) != null);
+        assertNotNull(s.getOption(IP_MULTICAST_IF));
 
         SocketAddress target = new InetSocketAddress(group, s.getLocalPort());
         long nano = System.nanoTime();
-        String text = nano + ": hello";
+        String text = nano + ": testSendNoReceive";
         byte[] message = text.getBytes("UTF-8");
 
         // send datagram to multicast group
```

Comment:
In the code below it would also be good to log the unexpected message content, as a string (or possibly both as a plain string and as an hexadecimal string.